services/provider/server/server.go

```diff
@@ -654,13 +655,24 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 // ReportStatus rpc call to check if a consumer can reach to the provider.
 func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStatusRequest) (*pb.ReportStatusResponse, error) {
 	// Update the status in storageConsumer CR
-
-	if _, err := semver.Parse(req.ClientPlatformVersion); err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "Malformed ClientPlatformVersion: %v", err)
+	klog.Infof("received client status: %s", req)
```

Comment:
Please print the ReportStatusRequest as an object and may start the message with the subject, not the verb 
```suggestion
	klog.Infof("Client status report received: %+v", req)
```

Reply:
> as an object (%s vs %+v)
- I don't think, this will make any diff as there is a string method already defined on this struct and both will log the same.

Reply:
done.